sites/hashdev/src/_pages/blog/4_announcing-error-stack.mdx

```diff
@@ -254,7 +252,7 @@ We haven't made it easy to create a new `Report` from a `String` because it requ
 
 **By consciously imposing more restrictions in some areas we hope to reduce the _overall_ cost of development, in keeping with the philosophy underpinning Rust itself.**
 
-## Some other cool things about `error-stack` that we don't have time for!
+### Some other cool things about `error-stack`
```

Comment:
```suggestion
### Some other cool things about error-stack
```
We're getting rid of code-formatted text in headings until we've managed to update the design to handle them better